src/Compilers/VisualBasic/Desktop/CommandLine/CommandLineParser.vb

```diff
@@ -1388,6 +1394,19 @@ lVbRuntimePlus:
                           End Function)
         End Function
 
+        Private Function ParseAnalyzerDependencies(name As String, value As String, diagnostics As IList(Of Diagnostic)) As IEnumerable(Of CommandLineAnalyzerDependency)
+            If String.IsNullOrEmpty(value) Then
+                ' TODO: localize <file_list>?
```

Comment:
Looks like we don't localize any of the strings in these error (e.g. ":<resinfo>" below). Probably worth a bug?